.github/CONTRIBUTING.txt

```diff
@@ -302,6 +302,72 @@ scikit-image/tools/deploy-docs.sh
 https://help.github.com/articles/creating-an-access-token-for-command-line-use/
 http://docs.travis-ci.com/user/encryption-keys/
 
+Deprecation cycle
+-----------------
+
+If the behavior of the library has to be changed, a deprecation cycle must be
+followed to warn users.
+
+- a deprecation cycle is *not* necessary when
+    * adding a new function, or
+    * adding a new keyword argument to the *end* of a function signature, or
+    * fixing what was buggy behaviour
+
+- a deprecation cycle is necessary for *any breaking API change*, meaning a
+    change where the function, invoked with the same arguments, would return a
+    different result after the change. This includes:
+    * changing the order of arguments or keyword arguments, or
+    * adding arguments or keyword arguments to a function, or
+    * changing a function's name or submodule, or
+    * changing the default value of a function's arguments.
+
+Usually, our policy is to put in place a deprecation cycle over two releases.
+
+For the sake of illustration, we consider the modification of a default value in
+a function signature. In version N (therefore, next release will be N+1), we
+have
+
+.. code-block:: python
+
+    def a_function(image, rescale=True):
+        out = do_something(image, rescale=rescale)
+        return out
+
+
+that has to be changed to
+
+.. code-block:: python
+
+    def a_function(image, rescale=None):
+    	if rescale is None:
+    	    warn('The default value of rescale will change to `False` in version N+3')
+    	    rescale=True
```

Comment:
Ach, sorry for being a pain, but let's make this code be PEP8-compatible: `rescale = True` (with spaces)